components/server/ee/src/prebuilds/prebuild-manager.ts

```diff
@@ -228,4 +238,33 @@ export class PrebuildManager {
         }
         return hostContext.contextParser;
     }
-}
\ No newline at end of file
+
+    private async shouldRateLimitPrebuild(span: opentracing.Span, cloneURL: string): Promise<boolean> {
+        const minuteAgo = secondsBefore(new Date().toISOString(), 60)
```

Comment:
This magic number is burried rather deep in here. It's ok to make this a constant if:
 - we move it more to the top, maybe by making it a unit-level `const` with a self-explanatory name
 - it's easily discoverable when looking for the config field. e.g., by adding a comment to the field description

Reply:
Updated.